src/main/java/ch/njol/skript/effects/EffStopServer.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * This file is part of Skript.
+ *
+ * Skript is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Skript is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.effects;
+
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+
+@Name("Stop Server")
+@Description("Stop/Restart the server. If restarting isn't configured, the server will stop.")
```

Comment:
What do you mean by "If restarting isn't configured, the server will stop."?

Reply:
![image](https://user-images.githubusercontent.com/42403698/88659607-9d117580-d0d5-11ea-81ae-1abdb9cbdb3e.png)

Reply:
what about 'if the restart-script spigot.yml option isn't set up the server will stop.'

Reply:
Sure

Reply:
Just this and then I think we can merge.